spec/structuraid_core/design_codes/schemas/rc/footings/bending_reinforcement_ratio_schema_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'spec_helper'
+
+RSpec.describe StructuraidCore::DesignCodes::Schemas::RC::Footings::BendingReinforcementRatioSchema do
+  describe '.validate!' do
+    subject(:result) { described_class.validate!(params) }
+
+    let(:params) do
+      {
+        design_compression_strength: 28,
+        design_steel_yield_strength: 420,
+        width: 2500,
+        effective_height: 450,
+        flexural_moment: 2_348_493_750,
```

Comment:
uugh, that's the ugly thing of using these units. Moments order of magnitude are too long

Reply:
hahahha yeah!